src/handle_query_contract_ui.c

```diff
@@ -1,42 +1,64 @@
 #include "figment_plugin.h"
 
-// Set UI for the "Amount" screen.
-static void set_amount_ui(ethQueryContractUI_t *msg) {
-    const char *title = "Amount";
-
+static void set_screen_title(ethQueryContractUI_t *msg, const char *title) {
     if (strlcpy(msg->title, title, msg->titleLength) < strlen(title)) {
         PRINTF("Screen title truncated: %s\n", msg->title);
     }
+}
+
+static void set_screen_message(ethQueryContractUI_t *msg, const char *message) {
+    if (strlcpy(msg->msg, message, msg->msgLength) < strlen(message)) {
+        PRINTF("Screen message truncated: %s\n", msg->msg);
+    }
+}
+
+// Set UI for the "Amount" screen.
+static bool set_amount_ui(ethQueryContractUI_t *msg) {
+    set_screen_title(msg, "Amount");
 
     const uint8_t *eth_amount = msg->pluginSharedRO->txContent->value.value;
     uint8_t eth_amount_size = msg->pluginSharedRO->txContent->value.length;
 
     // Converts the uint256 number located in `eth_amount` to its string representation and
     // copies this to `msg->msg`.
-    amountToString(eth_amount, eth_amount_size, WEI_TO_ETHER, "ETH", msg->msg, msg->msgLength);
+    return amountToString(eth_amount,
```

Comment:
Why is this camelCase and other functions snake_case?

Reply:
No idea 😄 This function is imported from `ethereum-plugin-sdk`: https://github.com/LedgerHQ/ethereum-plugin-sdk/blob/4d8e04480530cd14dd670a0a6405bca992d16942/include/eth_internals.h#L108-L113